lesson14/script.js

```diff
@@ -0,0 +1,95 @@
+const showLoadingImg = () => {
+    const div        = document.createElement('div');
+    const loadingImg = document.createElement('img');
+    div.id           = "js-loading";
+    loadingImg.src   = "img/loading-circle.gif";
+    loadingImg.alt   = "ローディング画像";
+    document.body.insertAdjacentElement('afterbegin',div).appendChild(loadingImg);
+}
+
+const removeLoadingImg = () => {
+    document.getElementById('js-loading').remove();
+}
+
+const renderList = (result) => {
+    const fragment = document.createDocumentFragment();
+    for(const list of result){
+        const li  = document.createElement('li');
+        const a   = document.createElement('a');
+        const img = document.createElement('img');
+        li.className  = "list-item";
+        a.href        = `/${list.a}`;
+        a.textContent = list.text;
+        img.src       = list.img;
+        img.alt       = list.alt;
+        fragment.appendChild(li).appendChild(a).insertAdjacentElement('afterbegin',img);
+    } 
+    document.getElementById('js-list').appendChild(fragment);
+}
+
+const fetchdata = async () => {
```

Comment:
```suggestion
const fetchData = async () => {
```
nits
It is very detailed but should be consistent with the Camel case.

Reply:
Thank you for teaching me!
I changed to a camel case.
https://github.com/nakagawamai/js-lesson/pull/16/commits/5e9cffb4b66a9a43a4f3b5e7613cc2126382ec23